serpent-moltres/isotope-distribution/notes.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
What's this file? It's not described in your PR

Reply:
True, I added the description in the README.
It calculates the homogeneous material density